support/Pipelines/Bbh/Inspiral.py

```diff
@@ -55,6 +56,77 @@ def inspiral_parameters(
     }
 
 
+def _load_spec_id_params(id_params_file: Path) -> dict:
+    """Load SpEC initial data parameters from 'ID_Params.perl'."""
+    # Here we have to deal with SpEC storing the initial data parameters in a
+    # perl script (yay!). We evaluate the perl script, emit the variables as
+    # YAML, and load that into Python as a dictionary.
```

Comment:
Since we are technically added a dependency on perl now, should we document that somewhere? I'm not sure if it's safe to assume that all systems will have perl in the default PATH (though they should...)